pkg/reconciler/revision/controller.go

```diff
@@ -105,9 +103,15 @@ func newControllerWithOptions(
 		return controller.Options{ConfigStore: configStore}
 	})
 
-	resolver := newBackgroundResolver(logger, &digestResolver{client: kubeclient.Get(ctx), transport: transport}, impl.EnqueueKey)
-	resolver.Start(ctx.Done(), 100)
+	transport := http.DefaultTransport
+	if rt, err := newResolverTransport(k8sCertPath, digestResolutionWorkers, digestResolutionWorkers); err != nil {
```

Comment:
Can we not override these values on the `transport` variable to avoid the passthrough and also have the benefits on the off chance that we actually fall back to the default here?

Reply:
Mmm maybe, but then we'll have the construction split between two places, which is kind of odd (I think I expect to see everything about the construction of the resolver transport in newResolverTransport).

Possibly the real question is why we have this fallback stuff 🤔.

Reply:
Indeed, maybe we should just quit :shrug:. Question for another day.